site/README

```diff
@@ -0,0 +1,25 @@
+# Building the website locally
+
+The website for the workshop, https://unidata.github.io/pyaos-ams-2021/index.html, is generated from Markdown using Sphinx.
+If you are using conda, you can follow these steps while inside the `site/` directory to build the website locally:
+
+1. Create the conda environment:
+   ```
+   conda env create -f site-environment.yml
+   ```
+
+2. Activate the environment:
+   ```
+   conda activate pyaos-ams-2021-site
+   ```
+
+3. Build the docs
+  - in a unix/mac environment:
+    `make html`
+  - in windows environment:
+    `make.bat html`
+
+The site will be located in the `build/html/` directory.
+Open `build/html/index.html` in your browser and stand back in awe!
+
+To clean up the generate files, run `make clean` or `make.bat clean`.
```

Comment:
Doesn't `make` on Windows find `make.bat`?

Reply:
It does, but won't execute by default in PowerShell:

~~~pwsh
> make clean
make: The term 'make' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.

Suggestion [3,General]: The command make was not found, but does exist in the current location. PowerShell does not load commands from the current location by default. If you trust this command, instead type: ".\make". See "get-help about_Command_Precedence" for more details.
~~~

Now, you can use `./make clean` or `.\make clean`, but at that point I use `ma<TAB><TAB><TAB><TAB> clean`, so we get the full `.\make.bat clean` (although in the commands above I've left out the starting `.\`, and that does not work). I'll update to include the `.\` and drop the `.bat`.